app/components/sections/hero-section.tsx

```diff
@@ -23,6 +29,23 @@ export function HeroSection({ children, title, body }: Props) {
 
   return (
     <>
+      {heroWithShapes && (
+        <motion.img
+          src={RainDrops}
```

Comment:
Let's just reference the source here instead of importing it. You'll need to omit `public` in this case. 
```suggestion
          src="/images/rain-drops-background.svg"
```